### PR TITLE
Adding the developer resources from https://www.womenwhocode.com/resources

### PR DESCRIPTION
**Pull request by thedamian**
If it's morally correct I'd like to add the amazing developer resources from Beginners to Advanced from the WomenWhoCode.com/resources webpage.

### CODE REVIEW

**Comment by thedamian** [comment]
Put ID's in Order
Changed some fields names 
Added the best  meetup..com groups around.

File: src/apiList.js [hunk 1]
```diff
@@ -197,15 +197,27 @@ module.exports = [
...
-    id: 20,
-    title: "Cartoons",
-    longDesc: "If cartoons is what you like then boy do we have a full list of all the cartoons from the past and present and all their details including a amazingly sourced image to showcase",
-    desc: "A list of Cartoons from your past.",
-    link: "cartoons",
-    graphLink: "cartoons/graphql",
-    endPoints: ["cartoons2D", "cartoons3D",]
-  },
+    endPoints: ["the-states"],
+    sampleCode: " \n fetch('https://sampleapis.com/the-states/api/the-states')\n .then(response => response.json())\n      .then(data =>  {\n        data.map(c => \n        {\n         console.log(c.title);\n        }\n      );\n  });"
```
Comment (jermbo) [inline src/apiList.js:201]: This sample code is no longer needed.

File: src/apiList.js [hunk 2]
```diff
@@ -197,15 +197,27 @@ module.exports = [
...
+   },
+   {
+	id: 20,
+	title: "Cartoons",
+	longDesc: "If cartoons is what you like then boy do we have a full list of all the cartoons from the past and present and all their details including a amazingly sourced image to showcase",
+	desc: "A list of Cartoons from your past.",
+	link: "cartoons",
+	graphLink: "cartoons/graphql",
+	endPoints: ["cartoons2D", "cartoons3D",],
+	sampleCode: " \n fetch('https://sampleapis.com/cartoons/api/cartoons2D')\n .then(response => response.json())\n .then(data => {\n data.map(c => \n {\n console.log(c.title);\n }\n );\n });"
```
Comment (jermbo) [inline src/apiList.js:211]: This sample code is no longer needed.

File: src/apiList.js [hunk 3]
```diff
@@ -197,15 +197,27 @@ module.exports = [
...
+  },
+  {
+    id: 21,
+    title: "Coding Resources",
+    longDesc: "Women Who Code.com is an amazing community and organization. They have an amazing resource in https://www.womenwhocode.com/resources. Here you'll find their 170 resources in an API for easy search, list and share. Please give a link BACK to WomenWhoCode.com if you use this information.",
+    desc: "API for all coding resources from https://www.womenwhocode.com/resources",
+    link: "codingResources",
+    graphLink: "codingResources/graphql",
+    endPoints: ["codingResources",], 
+    sampleCode: " \n fetch('https://sampleapis.com/codingResources/api/codingResources')\n .then(response => response.json())\n .then(data => {\n data.map(c => \n {\n console.log(c.url);\n }\n );\n });"
```
Comment (jermbo) [inline src/apiList.js:221]: This sample code is no longer needed.

**Comment by jermbo** [comment]
The backup file needs to be `codingResources.json.backup`. The refresh functionality is dependent on that naming convention.

File: src/apiList.js (hunk 1 above)
Comment (thedamian) [reply src/apiList.js]: OUTDATED!!! :date:

File: src/apiList.js (hunk 2 above)
Comment (thedamian) [reply src/apiList.js]: OUTDATED!!! :date:

File: src/apiList.js (hunk 3 above)
Comment (thedamian) [reply src/apiList.js]: OUTDATED!!! :date:

**Comment by thedamian** [comment]
File naming issue resolved

**Comment by thedamian** [comment]
sampleCode issue resolved

**Pull request merged**